cmd/kube-proxy/app/options.go

```diff
@@ -328,6 +315,32 @@ func (o *Options) processV1Alpha1Flags(fs *pflag.FlagSet) {
 	if fs.Changed("cluster-cidr") {
 		o.config.DetectLocal.ClusterCIDRs = strings.Split(o.clusterCIDRs, ",")
 	}
+	if fs.Changed("healthz-bind-address") {
```

Comment:
commented on the other PL, I think this is changing the existing behaviors of kube-proxy that are only configured via flags